src/main/java/org/testeditor/fixture/web/WebDriverFixture.java

```diff
@@ -822,5 +824,28 @@ protected void wrappedSleep(long ms, String msg, Map<String, Object> keyValueMap
             throw new FixtureException(msg, keyValueMap, e);
         }
     }
+    
+    /**
+     * A special keyboard key is pressed.
+     * 
+     * @param specialKey
+     *            the key to press (@see org.openqa.selenium.Keys)
+     * @throws FixtureException
+     *             if key is invalid
+     */
+    @FixtureMethod
+    public void pressSpecialKey(String specialKey) 
+            throws FixtureException {
+        if (specialKey == null || specialKey.trim().isEmpty()) {
+            throw new FixtureException("Invalid or empty key!");
+        }
 
+        try {
+            Keys seleniumKey = Keys.valueOf(specialKey.trim().toUpperCase());
+            new Actions(driver).sendKeys(seleniumKey).build().perform();
+        } catch (IllegalArgumentException e) {
+            throw new FixtureException("The specified key '" + specialKey.trim().toUpperCase()
```

Comment:
please do something like
new FixtureException("Key cannot be converted", FixtureException.keyValues("key", specialKey, "allowed-keys", Keys.values.join(", ")), e)